src/dmglib.py

```diff
@@ -213,6 +248,59 @@ def dmg_already_attached(path: str) -> bool:
     return os.path.realpath(path) in attached_images()
 
 
+def get_dmg_mountpoints(path: str) -> dict:
+    """Returns mountpoints of the already attached dmg.
+
+    Args:
+        path: path to the already attached disk image.
+
+    Returns:
+        Dict with mountpoints.
+
+    Raises:
+        InvalidOperation: if image is not already attached.
+    """
+    if not dmg_already_attached(path):
+        raise InvalidOperation()
+
+    success, infos = _hdiutil_info()
+
+    image = next(
+                filter(
+                    lambda image:
+                        'image-path' in image
+                        and image['image-path'] == os.path.realpath(path),
+                    infos.get('images', []),
+                )
+            )
+
+    return [entite['mount-point']
+            for entite in image.get('system-entities', [])
+            if 'mount-point' in entite]
+
+
+def detach_already_attached(path: str, force=True):
```

Comment:
Currently, method names start with `dmg_` mostly. Would renaming this method to `dmg_detach_already_attached` make sense to you, to stay consistent?